tests/functionnal/jenkins/bootstrap.sh

```diff
@@ -2,10 +2,6 @@
 
 set -x
 
-echo "deb http://http.debian.net/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list
-apt-get update && apt-get install -y -t jessie-backports openjdk-8-jre
```

Comment:
Don't we need:
`apt-get update && apt-get install -y openjdk-8-jre`
for Jenkins's slave?

Reply:
yes, we must. Let me fix that.